build/lib/pyaadhaar/decode.py

```diff
@@ -11,46 +12,51 @@
 class AadhaarSecureQr:
     # This is the class for Adhaar Secure Qr code..  In this version of code the data is in encrypted format
     # The special thing of this type of QR is that we can extract the photo of user from the data
+    # This class now supports 2022 version of Aadhaar QR codes [version-2]
     # For more information check here : https://103.57.226.101/images/resource/User_manulal_QR_Code_15032019.pdf
 
     def __init__(self, base10encodedstring):
         self.base10encodedstring = base10encodedstring
-        self.details = ["referenceid", "name", "dob", "gender", "careof", "district", "landmark",
-                        "house", "location", "pincode", "postoffice", "state", "street", "subdistrict", "vtc"]
-        self.delimeter = []
+        self.details = ["version","email_mobile_status","referenceid", "name", "dob", "gender", "careof", "district", "landmark",
+                        "house", "location", "pincode", "postoffice", "state", "street", "subdistrict", "vtc", "last_4_digits_mobile_no"]
```

Comment:
According to the docs, in the xml there should not be any field for last 4 digits of mobile no. Only mobile no hash will be there to verify with mobile no.
There is a reference id which is 4 digit of your aadhar card no

Reply:
But, while testing it with the new aadhaar cards I found the last 4 digit number is displayed there. These aadhaar cards are the newest ones, though there hasn't been any update in the documentation

Reply:
please check again, this 4digit nuber should be last 4 digiti of aadhar card. Check and confirm please